### ISSUE TITLE
Leave current window's tab alone

### ISSUE THREAD

**Issue by wrgrant** [gitobsidiantutorial/obsidian-tabs#31]
Currently, if you have a couple tabs open, when you activate a window, it removes that tab from the tab bar and replaces it with another one.

This is very clunky UX for anyone used to a web browser, as it requires you to remember which tab you have open, and where in the order of tabs it lives.

Could there be an option to leave the active tab alone in the tab bar (not replace it with one of the hidden tabs)?

**Comment by curtgrimes** [comment]
I agree. Because of this the tabs do not behave the way I would expect it to (like they would in a web browser's tabs).

**Comment by DaveyUS** [comment]
This is a must have imo. I'd imagine the solution might be a bit hacky and would require opening a second copy of the active note so the 'tab' could stay in place. If the dev is still active, I'd gladly sponsor to see this feature come to life!
